frontend_react/src/container/About.jsx

```diff
@@ -1,8 +1,130 @@
+  secondList.splice(abouts.length / 2, abouts.length - 1);
+
+  const handleClick = (title) => {
+    setSelectedItem(title);
+  };
+
+  useEffect(() => {
+    const query = '*[_type == "abouts"]';
+
+    client.fetch(query).then((data) => setAbouts(data));
+
+    console.log(abouts);
+  }, []);
 
-function About() {
   return (
-    <div>About</div>
+    <div id="ABOUT" className="md:h-viewport-height pt-[110px] overflow-y-auto">
+      <motion.div
+        initial={{ opacity: 0 }}
+        whileInView={{ opacity: [0, 1] }}
+        hidden={{ opacity: 1 }}
+        viewport={{ once: false, amount: 0.25 }}
+        className="flex flex-col pb-5 lg:flex-row md:px-[100px] lg:px-[200px] justify-evenly items-center gap-10 md:gap-20"
+      >
+        <div className="flex flex-col md:flex-row items-center flex-1 md:gap-2">
+          <div className="flex w-[86px] h-0.5 bg-primary mb-2" />
+          <div className="flex text-[36px] md:text-[42px] font-medium gap-2">
+            <span>My</span>
+            <br />
+            <span>Technologies</span>
+          </div>
+        </div>
+        <div className="items-center flex flex-row md:flex-col overflow-auto justify-start w-full">
+          <div className="flex flex-col max-h-[140px] md:flex-row justify-between items-center p-2 md:gap-10 gap-5 flex-1 ">
+            {firstList.map((about, index) => (
+              <motion.a
+                key={about.title + index}
+                whileHover={{ scale: 1.1 }}
+                href={about.link}
+                target="_blank"
+                rel="noreferrer"
+              >
+                <div className=" text-base font-medium justify-between items-center ">
+                  <img
+                    className="shadow-md shadow-primary w-11 h-11 md:w-10 md:h-10 object-cover items-center rounded-full mb-2"
+                    src={urlFor(about.imgUrl)}
+                    alt={about.title}
+                  />
+                  {about.title}
+                </div>
+              </motion.a>
+            ))}
+          </div>
+          <div className="flex flex-col max-h-[140px] md:flex-row justify-between items-center p-2 md:gap-10 gap-5 flex-1">
+            {secondList.map((about, index) => (
+              <motion.a
+                key={about.title + index}
+                whileHover={{ scale: 1.1 }}
+                href={about.link}
+                target="_blank"
+                rel="noreferrer"
+              >
+                <div className=" text-base font-medium justify-between items-center ">
+                  <img
+                    className="shadow-md shadow-primary w-11 h-11 md:w-10 md:h-10 object-cover items-center rounded-full mb-2"
+                    src={urlFor(about.imgUrl)}
+                    alt={about.title}
+                  />
+                  {about.title}
+                </div>
+              </motion.a>
+            ))}
+          </div>
+        </div>
```

Comment:
Is this for displaying the technologies you work with? 

If yes, you don't have to create firstList and secondList. Use flex with flex-wrap, which will automatically move elements to new rows depending on the screen size. 

Also, do not use max-h; giving a fixed value will make your content overflow/cut.